### DISCUSSION TITLE
Auto3DSeg vs. nnUNet vs. DynUNet

### DISCUSSION THREAD

**Discussion by landynfrancis** [Project-MONAI/tutorials#1711] [General] [answered]
Hello all,

I have been using pieces of MONAI for a while now to make basic 3D Segmentation models primarily with UNET. I have since noticed other pipeline/frameworks such as Auto3DSeg, nnUNET and DynUNET. I am interested in starting to use these, but I wanted to ask about Auto3DSeg beforehand. It seems that Auto3DSeg might just be a MONAI specific version of nnUNET and DynUNET, is this correct or is there more to it?

**Comment by KumoLiu** [comment, marked as answer]
Hi @landynfrancis, instead of a MONAI specific version of nnUNET and DynUNET, Auto3DSeg is a self-contained solution for 3D medical image segmentation which contain several algorithms.
More details, you can refer to:
https://github.com/Project-MONAI/tutorials/tree/main/auto3dseg

Thanks.

**Reply by landynfrancis** [reply]
Thank you, I will read the documentation pages.
